dpdispatcher/machine.py

```diff
@@ -124,6 +125,13 @@ def load_from_json(cls, json_path):
         machine = cls.load_from_dict(machine_dict=machine_dict)
         return machine
 
+    @classmethod
+    def load_from_yaml(cls, yaml_path):
+        with open(yaml_path) as f:
+            machine_dict = yaml.safe_load(f)
+        machine = cls.load_from_dict(machine_dict=machine_dict)
```

Comment:
Please call `load_from_dict` (line 136) instead. It can check whether the type is correct. Same for `Resources` and `Task`.

Reply:
sorry, what do you mean by "call load_from_dict"? I follow the same code of load_from_json.

Reply:
Sorry, I found `Machine.load_from_yaml` is correct...

Reply:
No worries. I was concerning if some problems from my side